Pocket/Pocket/Pocket.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Pocket.swift
+//  Pocket
+//
+//  Created by Luis De Leon on 6/5/18.
+//  Copyright © 2018 Pocket Network. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+public typealias TransactionHandler = (_: TransactionResponse?, _: Error?) -> Void
+public typealias ExecuteQueryHandler = (_: QueryResponse?, _:Error?) -> Void
+
+class Pocket {
+    var requestManager:PocketRequestManager?
+    static var pocketInstance:Pocket?
+    var pocketNodeURL:URL?
+    
+    class func getInstance(pocketNodeURL: URL) -> Pocket {
+        if(pocketInstance == nil) {
```

Comment:
use a `guard` statement instead of a nil check. Also, `if` statements in Swift do not require parentheses.